src/pages/Authors/Authors.js

```diff
@@ -42,13 +42,16 @@ export default function Authors(){
     }
   ].sort(() => Math.random() - 0.5)
 
+  const AllAuthors = () => {
+    return authorsBio.map(bio => <AuthorBio key={bio.name} bio={bio} />)
+  }
```

Comment:
One thing I did was pull these `.map` function calls out into their own private Component. It allows us to call it in the code like any other component (`<AllAuthors />`), it also allows you to encapsulate this code outside the view. This code is pretty simple right now, but the complexity could grow quickly and its nice to start with it already untangled from the view code.